src/index.js

```diff
@@ -301,6 +287,11 @@ export default function(api) {
         let skippedIdentifiers = 0
         const removeNewlyUnusedIdentifiers = {
           VariableDeclarator(path) {
+            // Only consider the top level scope.
+            if (path.scope.block.type !== 'Program') {
```

Comment:
```js
const referencedPropTypes = {
   variant: () => {
    const willBeWrapped = 1; // will still be wrapped, no?
    return null;
  }
}

Component.propTypes = { ...referencedPropTypes }
```

That would still wrap `willBeWrapped`, no?

Reply:
Let me add a test

Reply:
Not sure where my brain was. `willBeWrapped` is obviously not in the program scope.